docs/source/en/_toctree.yml

```diff
@@ -88,6 +88,11 @@
     - local: generation_strategies
       title: Customize the generation strategy
     title: Generation
+  - isExpanded: false
+    sections:
+    - local: tasks/idefics
+      title: Image tasks with IDEFICS
+    title: Prompting
```

Comment:
Seems to be the only task with a model name (`/tasks/idefics` , not very intuitive for me but since everyone seems okay with this no problem! 😉

Reply:
Yes, it's in its own "Prompting section" which will also later have a guide on prompting LLMs for different tasks. So it's still about tasks, but with a different approach. prompting vs finetuning.